compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -147,37 +149,24 @@ bool is_quantized(const CircleNode *node)
          node->dtype() == loco::DataType::S32;  // bias
 }
 
-void asym_wquant_with_minmax(CircleConst *node, float min, float max, float *scaling_factor,
-                             int64_t *zp)
+void asym_wquant_with_minmax(CircleConst *node, float min, float scaling_factor)
```

Comment:
It would be better to change the function name.

Maybe something like `asym_wquant_with_min_scale` or simply `asym_wquant`.

I'd preferred the second one.

Reply:
I also agree the second one :-) I will apply it.